include/mockturtle/traits.hpp

```diff
@@ -545,6 +546,21 @@ template<class Ntk>
 inline constexpr bool has_create_node_v = has_create_node<Ntk>::value;
 #pragma endregion
 
+#pragma region has_create_cover_node
+template<class Ntk, class = void>
+struct has_create_cover_node : std::false_type
+{
+};
+
+template<class Ntk>
+struct has_create_cover_node<Ntk, std::void_t<decltype( std::declval<Ntk>().create_node( std::declval<std::vector<signal<Ntk>>>(), std::declval<std::pair<std::vector<kitty::cube>, bool>>() ) )>> : std::true_type
```

Comment:
It's confusing that you check if a network type `has_create_cover_node` but the function name is still `create_node` (with different argument types). I would suggest to call this function really `create_cover_node` to distinguish.